src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Commands/ReportCommand.cs

```diff
@@ -38,6 +61,8 @@ internal async Task<int> InvokeAsync(
                     cancellationToken: cancellationToken).ConfigureAwait(false))
             {
                 results.Add(result);
+
+                logger.LogInformation("Execution: {executionName} Scenario: {scenarioName} Iteration: {iterationName}", result.ExecutionName, result.ScenarioName, result.IterationName);
```

Comment:
minor: This could end up being quite verbose when there are 1000s of scenarios and iterations. Should we limit this logging to verbose mode (and introduce a flag for verbosity)?

Reply:
It doesn't seem that bad to me. I added it because there is no other indication of progress when generating the report and it can take more than a few seconds.

I really would like to replace the logger. Maybe it's just me, but the output is fairly unreadable I think. Would like to have something more like the new dotnet build logging, that can update interactively. I'm open to updates here, but I feel like this is at least an improvement for now.

Reply:
Sounds good - interactive updates that don't scroll the output would be great for the future.